docs/career_assessment_how_to.md

```diff
@@ -0,0 +1,65 @@
+# Career assessment
+## How does it work?
+Each "Me" or "Not Me" response is matched with a trait like "Adaptable" or "Organized". Each career is associated with a few traits. When results are calculated, each career is given a score to reflect how closely it matches the traits chosen by the job seeker. The three careers with the highest score are displayed.
+
+Matching an image with a trait can happen three ways:
```

Comment:
Perhaps organizing the 3 traits in list bullets to make it more legible? <ul> Matching an image with a trait can happen three ways:
1.
2.
3.